### ISSUE TITLE
No error message with invalid password logging into server

### ISSUE THREAD

**Issue by aerwin** [maqetta/maqetta#3230]
Using Chrome on Mac going to Moketta...

Enter wrong password and get no error indication. The following is in the console:

```
Failed to load resource: the server responded with a status of 401 (Unauthorized) 
```

**Comment by jhpedemonte** [comment]
-> @childsb.  cc @JonFerraiolo

**Comment by jhpedemonte** [comment]
The error message seems to be hidden in the login screen.  Login with an invalid password or username, then click on "Register".  Notice the "Invalid user or password" alert below the register pane.

**Comment by childsb** [comment]
Fixed with checkin cfec354f6eebf5ea86b36da06da63c9c9fc812d4
